pbiviz.json

```diff
@@ -1,10 +1,10 @@
 {
   "visual": {
     "name": "MekkoChart",
-    "displayName": "Mekko Chart 1.9.3",
+    "displayName": "Mekko Chart 1.9.4 Preview",
     "guid": "MekkoChart1449744733038",
     "visualClassName": "MekkoChart",
-    "version": "1.9.3",
+    "version": "1.9.4",
```

Comment:
Do unit tests cover changes that you made?

Reply:
Exist tests cover changes